views/widget-front.php

```diff
@@ -71,7 +71,7 @@ class_exists( 'Statify' ) || exit;
 							<?php echo (int) $target['count']; ?>
 						</td>
 						<td class="t">
-							<a href="<?php echo esc_url( home_url( $target['url'] ) ); ?>" target="_blank" rel="noopener noreferrer">
+						<a href="<?php echo esc_url( home_url( $target['url'] ) ); ?>" target="_blank" rel="noopener noreferrer" title="<?php echo 0 == url_to_postid( $target['url'] ) ? esc_html( get_bloginfo( 'name' ) ) : esc_html( get_the_title( url_to_postid( $target['url'] ) ) ); ?>">
```

Comment:
[`esc_html`](https://developer.wordpress.org/reference/functions/esc_html/) escapes HTML code like `<`, `>` and some special characters.

You are populatione the `title` _attribute_ here, so slightly different escaping is required here, like quotes `"`.
E.g. `foo=" bar="test` passes HTML escaping, but the final code would be `title="foo" bar="test"` - obviously not quite the desired result.

We should use [`esc_attr`](https://developer.wordpress.org/reference/functions/esc_attr/) in this case.